test/cases_matching_callbacks/4.callbacks.c

```diff
@@ -0,0 +1,15 @@
+#include <metababel/metababel.h>
+#include <stdio.h>
+
+static void usr_foo_event_callback(void *btx_handle, void *usr_data) {
+  printf("foo\n");
```

Comment:
are this is used for validating?

Reply:
I have no idea on how to do something more simple than this :(

Reply:
I found a way, thank Thomas